### ISSUE TITLE
Tutorial: "Input and Access Types" doesn't list all available input and access types

### ISSUE THREAD

**Issue by hilmarf** [open-component-model/ocm-website#325]
https://ocm.software/docs/tutorials/input-and-access-types/ is incomplete

is this also generated or hand written?

**Comment by morri-son** [comment]
> https://ocm.software/docs/tutorials/input-and-access-types/ is incomplete
> 
> is this also generated or hand written?

@hilmarf the document has been created by @jensh007. It offers a good overview of the existing input and access types that is not available in that condensed form in the OCM spec. A disadvantage of such docs is that - as you recognised - that it needs to be double-maintained whenever the source is adopted, which is effort, hard to follow consistently and often fails. But I guess the same is true if we would move this doc to the spec itself as it collects information from various places in the spec. I think this document offers a very good overview and should be kept, but I agree that we have to find a way to keep it consistent with any change to referenced sources.

Do you have a proposal on how we can achieve that?
